runtime/common/marshaling/marshal_errors.ts

```diff
@@ -37,7 +37,7 @@ function getErrorClassType(err: Error): ErrorClassType {
   return ErrorClassType.Other;
 }
 
-export function marshalError(err: any): object | undefined {
+export function marshalError(err: any, marshalFn: (val: any) => any): object | undefined {
```

Comment:
I suppose `marshalValue` is the only possible value of `marshalFn`. Do we still need this parameter?

Reply:
This was just a lazy way to get around circular dependencies, since marshalValue depends on marshalError.

I'll just move the contents of marshal_errors.js into index.js for simpler code

Reply:
cool. thanks